packer/grapl-service/image.pkr.hcl

```diff
@@ -0,0 +1,163 @@
+packer {
+  required_version = ">= 0.12.0"
+}
+
+variable "build_ami" {
+  description = "Whether or not to actually build an AMI. Set to `false` for doing testing"
+  type        = bool
+  default     = true
+}
+
+variable "ami_name_prefix" {
+  type    = string
```

Comment:
It would be good to add `description` fields for every variable.

Reply:
done